### PR TITLE
OCPBUGS-17161 - hide footer after a few seconds

### PR DESCRIPTION
**Pull request by aireilly**
https://issues.redhat.com/browse/OCPBUGS-17161

The footer is currently obscuring content at the end of the page. The previous convoluted js solution to adjust the page is broken. Maybe some recent Chrome or HTML/CSS spec change. 

This is a pretty dumb solution, no css fades etc but it works :shrug: 

Demo: https://file.emea.redhat.com/aireilly/add-footer-transition/welcome/index.html#hosted-control-plane-activities

### CODE REVIEW

File: _templates/_page_openshift.html.erb [hunk 1]
```diff
@@ -20,7 +20,7 @@
   <link href="https://assets.openshift.net/content/subdomain/touch-icon-precomposed.png" rel="apple-touch-icon-precomposed" type="image/png"/>
   <link href="https://assets.openshift.net/content/subdomain/favicon32x32.png" rel="shortcut icon" type="text/css"/>
   <link href="https://assets.openshift.net/content/osh-nav-footer.css" rel="stylesheet" type="text/css" media="screen"/>
-  <link href="https://docs.openshift.com/container-platform/4.1/_stylesheets/docs.css" rel="stylesheet"  media="screen"/>
+  <link href="https://file.emea.redhat.com/aireilly/add-footer-transition/_stylesheets/docs.css" rel="stylesheet"  media="screen"/>
```
Comment (aireilly) [inline _templates/_page_openshift.html.erb:23]: Remember to put this back before merging.

**Review by wgordon17 — APPROVED** [review]
Good enough...given the current state of our CSS ➕1

**Comment by aireilly** [comment]
Merged in error. Work will be completed in https://github.com/openshift/openshift-docs/pull/73774

**Pull request merged**